src/test/test4/Dyn.java

```diff
@@ -0,0 +1,22 @@
+package test4;
+
+public class Dyn {
```

Comment:
What is the purpose of this class? Is it used somewhere?

Reply:
Removed the file